### ISSUE TITLE
Bug (Mac OS): No Notifications are appearing at all

### ISSUE THREAD

**Issue by chrisgrieser** [ramboxapp/community-edition#3043]
After installing Rambox, no notifications are displayed at all. Rambox' Dock Icon does display Badges, though. Checking in the Notification Settings of Mac OS, Rambox also seems to be missing:

<img width="668" alt="Screen Shot 2021-07-07 at 20 04 32" src="https://user-images.githubusercontent.com/73286100/124808319-13ae2880-df5f-11eb-9640-9defb38cc371.png">

Earlier this year, I already had installed Rambox on this very machine, with Notifications showing properly. Now re-installing Rambox, Notifications do not work anymore.

### ENVIRONMENT

> Version: 0.7.7
Platform: darwin (x64)
Electron: 7.2.4
Chromium: 78.0.3904.130
Node: 12.8.1

> OS: Mac OS 11.4 (Intel)

**Comment by fvulich** [comment]
That's odd, can you try uninstalling Rambox following these [steps](https://github.com/ramboxapp/community-edition/wiki/Clean-Uninstall), and install it again? Then, once you get the first notification, Rambox should appear in the notification center.

**Comment by chrisgrieser** [comment]
Ok, the "allow notifications" message now appeared and Rambox appears in the Mac OS Notifications settings.

However, some notifications are now unreliable, depending on the service: Facebook Messenger is spotty, and Telegram does not trigger any notifications, but the Unread Counter Badge (the red number at the Dock Icon) appears correctly. Whatsapp works fine without any issues.

Notifications are properly enabled in the Mac OS System Settings, and the Rambox settings of every service I use as well.

<img width="168" alt="Screen Shot 2021-07-16 at 22 07 40" src="https://user-images.githubusercontent.com/73286100/126003200-6ad22c48-59b2-4a8d-b40b-74106bf80496.png">

<img width="626" alt="Screen Shot 2021-07-16 at 22 06 24" src="https://user-images.githubusercontent.com/73286100/126003037-91d46983-8272-4049-a811-0daade006d4d.png">

**Comment by fvulich** [comment]
Is this still reproducible in 0.7.9?

**Comment by chrisgrieser** [comment]
yes, notifications are still spotty. Discord and WhatsApp work, but Telegram and Facebook Messenger do not send notifications at all.
